.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release New Version
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create new GitHub tag
```

Comment:
do we need to use the checkout GHA step first?

Reply:
Yes I do. I will add that if we end up going this route.